examples/lfric/eg12/extract_kernel_with_transformations.py

```diff
@@ -84,8 +86,6 @@ def trans(psy):
     # Get instance of the ExtractTrans transformation
     etrans = LFRicExtractTrans()
 
-    # Import transformation script and apply transformations
-    import colouring_and_omp as transformation
     psy = transformation.trans(psy)
```

Comment:
Note that this is LFRic, so still uses psy

Reply:
I see I had to move the transformation out for a subtle change, now the script is loaded but executed latter, while before it did both at the same place. This means that (in the first review) we removed the path of the script from the sys.path and the script can't find it when importing in the executable part (it still succeeded if they were at the top because then this happens at import time).

I would still expect the imports in the execution part to work, so I now deleted the part that removed the script path from the sys.path.